assets/js/book-manager.js

```diff
@@ -0,0 +1,90 @@
+const inputTitle = document.querySelector('.input-title');
+const inputAuthor = document.querySelector('.input-author');
+// const addButton = document.querySelector('.add-button');
+const bookShelf = document.querySelector('.book-shelf');
+const bookCard = document.createElement('div');
+
+const Book = function BookList(title, author) {
+  this.title = title;
+  this.author = author;
+
+  this.removeButton = function removeButton(element) {
+    if (element.classList.contains('remove-button')) {
+      element.parentElement.remove(element);
+    }
+  };
+
+  this.getToloStorage = function getToloStorage() {
+    let book;
+    if (!localStorage.getItem('books')) {
+      book = [];
+    } else {
+      book = JSON.parse(localStorage.getItem('books'));
+    }
+    return book;
+  };
+
+  this.addToLocalStorage = function addToLocalStorage(book) {
+    const books = this.getToloStorage();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  };
+
+  this.removeToLocalStorage = function removeToLocalStorage(id) {
+    let books = this.getToloStorage();
+    const tempBooks = [];
+    books.forEach((book, index) => {
+      if (index !== parseInt(id, 10)) {
+        tempBooks.push(book);
+      }
+    });
+    books = tempBooks;
+    localStorage.setItem('books', JSON.stringify(books));
+  };
```

Comment:
- [ ] There seems to be a bug in your app. I loaded your app and add some books, now when I delete books, they disappear from the page but not from the `localStorage` collection. This means that when I reload the page, the books I previously deleted still reappear. Kindly fix this.

  Books are added | One book deleted | Second book deleted
  :---:|:---:|:---:
  ![pasu-1st](https://user-images.githubusercontent.com/36542986/153160254-1fced8b1-1f5f-4798-864c-eaa99701278a.jpg) | ![pasu-2nd](https://user-images.githubusercontent.com/36542986/153160262-444cfcbb-f21d-42bb-a843-c9a4cddb1ca6.jpg) | ![pasu-3rd](https://user-images.githubusercontent.com/36542986/153160263-1f6e2062-6f53-467d-9d7f-0fadb70412f1.jpg)

Reply:
Hello @TedLivist,

Reply:
Hi @vikipretium, the live link still exhibits the same behaviour I highlighted